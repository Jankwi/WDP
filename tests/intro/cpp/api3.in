11
dziekuje gleboki 7
dziekuje plytki 9
prosze gleboki
dziekuje gleboki 11
dziekuje gleboki 2137
prosze gleboki
prosze plytki
dziekuje plytki 12
dziekuje plytki 12
prosze plytki
prosze plytki