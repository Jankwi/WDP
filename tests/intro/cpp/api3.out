7
2137
9
12
12